backend/src/main/scala/com/softwaremill/bootzooka/http/EndpointsToRoutes.scala

```diff
@@ -38,15 +40,19 @@ class EndpointsToRoutes(http: Http, apiContextPath: String) {
     * code, by translating it using the `http.exceptionToErrorOut` method and using that to create the response.
     */
   private val decodeFailureHandler: DecodeFailureHandler = {
-    def failResponse(defaultDecodeFailureResponse: DefaultDecodeFailureResponse, msg: String): DecodeFailureHandling =
-      DecodeFailureHandling.response(http.failOutput)((defaultDecodeFailureResponse.status, Error_OUT(msg)))
+    def failResponse: (StatusCode, List[Header], String) => ValuedEndpointOutput[_] =
+      (c, hs, m) => ValuedEndpointOutput(statusCode.and(headers).and(stringBody), (c, hs, m))
```

Comment:
we should try to reuse `http.failOutput` so that the shape of errors is always the same